src/Tmds.Ssh/SftpClient.cs

```diff
@@ -11,15 +11,28 @@ abstract class SftpOperation
     {
         public abstract ValueTask HandleResponse(SftpPacketType type, ReadOnlySequence<byte> fields, SftpClient client);
 
+        protected static Exception CreateExceptionForStatus(SftpErrorCode errorCode, string? errorMessage)
+        {
+            return new SshException(errorMessage ?? "No error message specified", errorCode);
+        }
         protected static Exception CreateExceptionForStatus(ReadOnlySequence<byte> fields)
         {
-            return new SshException("TODO"); // TODO: use fields
+            var statusTuple = ParseStatusFields(fields);
+            return new SshException(statusTuple.errorMessage ?? "No error message specified", statusTuple.errorCode);
```

Comment:
`"No error message specified"` -> `$"Error {errorCode}"`

Reply:
We should have a dedicated exception type `SftpException : SshException`.
The constructor should accept `(SftpErrorCode errorCode, string? errorMessage)`.